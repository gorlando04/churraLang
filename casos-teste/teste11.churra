amigo "Felipe" contribui 30
amigo "Gabriel" contribui 30
amigo "Caio" contribui 30
amigo "Heitor" contribui 30 vegetariano
amigo "Jorge" contribui 30
amigo "GIullio" contribui 30 vegetariano
amigo "Lucca" contribui 30
amigo "Ana" contribui 30
amigo "Roberta" contribui 30 vegetariano
amigo "Abbiati" contribui 30

item comida preco_unit 20 quantidade 5
item carne "alcatra" preco_kg 50 quantidade_kg 2
item entretenimento preco_unit 50 quantidade 1
item bebida preco_L 25 quantidade_L 2

rateio igual