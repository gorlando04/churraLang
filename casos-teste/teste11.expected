Total do churrasco: R$300.00
Abbiati deve pagar: R$34.29
Ana deve pagar: R$34.29
Caio deve pagar: R$34.29
Felipe deve pagar: R$34.29
GIullio deve pagar: R$20.00
Gabriel deve pagar: R$34.29
Heitor deve pagar: R$20.00
Jorge deve pagar: R$34.29
Lucca deve pagar: R$34.29
Roberta deve pagar: R$20.00